larq/testing_utils.py

```diff
@@ -2,9 +2,6 @@
 import numpy as np
 import tensorflow as tf
 
-# We should find a better solution without relying on private objects
-from tensorflow.python.keras.testing_utils import _thread_local_data, should_run_eagerly
```

Comment:
Nice, I was worried that this will just brake at some point. Glad we can get rid of it.